test/lbd-tx-summary-adapter.spec..ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from "chai";
```

Comment:
this filename and its extension looks wrong. maybe `lbd-tx-summary-adpater.spec.ts` looks right instead of `lbd-tx-summary-adpater.spec..ts`

Reply:
Oops my bad. :(